<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>307</width>
    <height>266</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Risk Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="fixedLabel">
      <property name="text">
       <string>This project analysis can determine the risks on a new project based on cost and personnel.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="analyseButton">
      <property name="text">
       <string>Analyse</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="costLabel">
      <property name="text">
       <string>Cost (in millions):</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="costSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="personnelSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="riskLabel">
      <property name="text">
       <string>Risk:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="personnelLabel">
      <property name="text">
       <string>Personnel:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="2">
     <widget class="QLabel" name="riskResultLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Press Analyse to determine a risk</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Clears the Label</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
